src/main/resources/config/en.yml

```diff
@@ -94,29 +99,37 @@ Experiment_MCDiscordReserializer_ToMinecraft: false
 Experiment_MCDiscordReserializer_InBroadcast: false
 
 # Other
+# Whether to cancel a console command if it can't be logged
 CancelConsoleCommandIfLoggingFailed: true
+# A language code to force DiscordSRV to use; See https://github.com/DiscordSRV/DiscordSRV/tree/master/src/main/resources/config for a list of available codes
 ForcedLanguage: none
+# Whether to print a list of guilds and their channels to the console on start
 PrintGuildsAndChannels: true
+# Whether to force the use of TLSv12
 ForceTLSv12: true
+# Returns valid hostname without making an attempt to actually validate it (insecure)
 NoopHostnameVerifier: false
```

Comment:
Comment is hard to understand, I'd suggest "Disables validation of hostnames (insecure)"